samples/create_folder_in_datacenter.py

```diff
@@ -65,10 +91,20 @@ def main():
         password = getpass.getpass(prompt='Enter password for host %s and '
                                    'user %s: ' % (args.host, args.user))
 
+    if args.nocert:
+        print("WARNING: SSL certificate check is skipped!")
+        s_context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
```

Comment:
I think we can use ssl._create_unverified_context() or connect.SmartConnectNoSSL() method if Cert validation is skipped

Reply:
@pgbidkar Sorry for long response awaiting.
I did that change. Please check if it works (as I mentioned above, I don't have VMware env).